intro-to-react/books-lisiting/src/types/book.ts

```diff
@@ -0,0 +1,27 @@
+import { Dispatch, SetStateAction } from "react";
+
+export type Book = {
+  author: string;
+  title: string;
+  year: number | string;
+};
+
+export interface BookItemProps {
```

Comment:
try to use naming conventions for interface and start all interface with I example IBookItemProps